extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/runtime/src/main/java/io/quarkus/resteasy/reactive/jackson/runtime/serialisers/JacksonMessageBodyWriter.java

```diff
@@ -79,8 +79,9 @@ public void writeResponse(Object o, Type genericType, ServerRequestContext conte
         } else {
             // First test the names to see if JsonView is used. We do this to avoid doing reflection for the common case
             // where JsonView is not used
-            if (context.getResteasyReactiveResourceInfo().getMethodAnnotationNames().contains(JSON_VIEW_NAME)) {
-                Method method = context.getResteasyReactiveResourceInfo().getMethod();
+            ResteasyReactiveResourceInfo resourceInfo = context.getResteasyReactiveResourceInfo();
```

Comment:
In which case can this be null? It seems a bit counter intuitive.

Reply:
It can be null in two legitimate cases:

1) An exception is thrown before the target method is reached
2) A request filters aborts the processing